Cubical/Data/Sigma/Properties.agda

```diff
@@ -193,3 +196,99 @@ PiΣ = isoToEquiv (iso (λ f → fst ∘ f , snd ∘ f)
 
 swapΣEquiv : ∀ {ℓ'} (A : Type ℓ) (B : Type ℓ') → A × B ≃ B × A
 swapΣEquiv A B = isoToEquiv (iso (λ x → x .snd , x .fst) (λ z → z .snd , z .fst) (\ _ → refl) (\ _ → refl))
+
+Σ-split-iso : ∀ {ℓ} {A : Set ℓ} {B : A → Set ℓ} {a a' : A} {b : B a} {b' : B a'} → Iso (Σ (a ≡ a') (λ q → PathP (λ i → B (q i)) b b')) ((a , b) ≡ (a' , b'))
+Iso.fun (Σ-split-iso) = ΣPathP
+Iso.inv (Σ-split-iso) eq = (λ i → fst (eq i)) , (λ i → snd (eq i))
+Iso.rightInv (Σ-split-iso) x = refl {x = x}
+Iso.leftInv (Σ-split-iso) x = refl {x = x}
+
+Σ-split : ∀ {ℓ} {A : Set ℓ} {B : A → Set ℓ} {a a' : A} {b : B a} {b' : B a'} → (Σ (a ≡ a') (λ q → PathP (λ i → B (q i)) b b')) ≡ ((a , b) ≡ (a' , b'))
+Σ-split = ua Σ≡
+
+Σ-split-iso' : ∀ {ℓ} {A B : Set ℓ} {a a' : A} {b' b : B} → (Σ (a ≡ a') (λ q → b ≡ b')) ≡ ((a , b) ≡ (a' , b'))
+Σ-split-iso' = ua Σ≡
+
+Σ-ap-iso₂ : ∀ {i j} {X : Set i}
```

Comment:
Why use i j for levels and not \ell?

Reply:
The same applies to the rest of the file. i and j are usually used for dimension variables

Reply:
Oh, I sometimes run into the problem that the UTF-8 part of agda mode in emacs does not work, so i have some places where this is an issue. I will fix it asap.